Fix docu link URL on compound configuration (#3730)
SAP/luigi: docs/luigi-compound-container-api.md

### compoundConfig

> The configuration for the compound microfrontend

Reviewer: .
```suggestion
The configuration for the compound microfrontend.
```

Just saw that a dot seem to have been missing here. You can just commit the change.

> We recommend you taking a look at the Docs covering the compund [here](https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=compound)

Reviewer: ```suggestion
Take a look at the [compound parameter](https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=compound) for details. 
```
This would suit better
